UnleashSampleApp/ViewController.swift

```diff
@@ -8,9 +8,13 @@
 import UIKit
 import Unleash
 
-// This constant is to emulate a client knowing its target environment is (e.g. QA, Production)
 struct Constants {
-  static let environment: String = "QA"
+  // This constant is to emulate a client knowing its target environment is (e.g. qa, stage, production)
+  static let environment: String = "qa"
+  static let appName = "ios-unleash-sample-app"
+  static let unleashUrl = "https://unleash.silvercar.com/api"
+  static let featureToggleName = "enable-ios-unleash-sample-app"
```

Comment:
Did you create a new toggle in Unleash?

Reply:
Yes sir I did. One specific for the sample app.